src/stores/state/Patch.ts

```diff
@@ -51,6 +52,15 @@ function add(pointerTarget: PointerTarget, value: any): any {
 	return pointerTarget.object;
 }
 
+function assert(pointerTarget: PointerTarget, value: any) {
```

Comment:
I moved it to match the other sections and because the test seemed like a separate responsibility from transversing the paths. Since I moved the comparison stuff for the difference message into its own function there's not a lot of complexity so I guess it's ok to move it back.